aptos-move/aptos-vm-types/src/change_set.rs

```diff
@@ -608,6 +607,12 @@ impl VMChangeSet {
                             &mut group_entry.get_mut().inner_ops,
                             additional_inner_ops,
                         )?;
+
+                        let maybe_op_size = group_entry
+                            .get_mut()
+                            .maybe_group_op_size
```

Comment:
We need to re-calculate the size based on group writes, no? For example, if I modify the resource member to become bigger, we should use the additional op size?

Reply:
each write to a resource group contains everything, and so if the second write contains the size it should be the up-to-date size at that point, right?

Reply:
Yes, no recalculation, but the size should be `additional_maybe_group_op_size`?

Reply:
oh... good point

Reply:
yeah this should be additional_maybe_group_op_size.unwrap()? since it can't be Deletion in this branch?